src/zkevm_specs/opcode/stack.py

```diff
@@ -48,4 +58,19 @@ def check_dupx():
     evm_stack.push(word2)
     assert(word2 == evm_stack.peek())
     evm_stack.dupx(2)
-    assert(word1 == evm_stack.peek())
\ No newline at end of file
+    assert(word1 == evm_stack.peek())
+
+@is_circuit_code
```

Comment:
move the test to `zkevm-specs/tests/test_opcode.py`

Reply:
udpated